tests/conftest.py

```diff
@@ -1,12 +1,13 @@
-import pytest
-import numpy as np
-
-import pennylane as qml
-from pennylane_qiskit import AerDevice, BasicAerDevice
-
 import contextlib
 import io
 
+import numpy as np
+import pennylane as qml
+import pytest
+from packaging import version
+from qiskit import __version__
```

Comment:
These two lines are newly added, the rest of the imports were reordered according to `isort`.